#!/usr/bin/env python
PACKAGE = "yahboomcar_laser"
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("ang_Kp", double_t, 0, "Kp in PID", 3.0, 0, 10)
gen.add("ang_Ki", double_t, 0, "Ki in PID", 0.0, 0, 10)
gen.add("ang_Kd", double_t, 0, "Kd in PID", 5.0, 0, 10)
gen.add("laserAngle", int_t, 0, "laserAngle", 70, 10, 90)
gen.add("ResponseDist", double_t, 0, "ResponseDist", 0.5, 0, 8)
gen.add("switch", bool_t, 0, "switch in rosbot", False)
exit(gen.generate(PACKAGE, "laser_Warning", "laserWarningPID"))
